events/events-team/events-lead.md

```diff
@@ -1,33 +1,84 @@
 # Events Lead Handbook
 
+The Event Lead is responsible for acting as a chair for the event, carrying out
+the strategic vision, and being the glue needed to execute on the event.
+
+- [Overview](#overview)
+- [Skills and Qualifications](#skills-and-qualifications)
+- [Activities](#activities)
+- [Scheduling and Communication Property Access](#scheduling-and-communication-property-access)
+
 ## Overview
 
-The Event Lead is responsible for acting as a chair for the event, carrying out the strategic vision, and being the glue needed to execute on the event.
+The Event Lead(s) is responsible for determining which core roles need to be
+filled at the start of planning for each event. After initial core team meetings,
+each lead will determine how to best staff for that function and fill in other
+roles. Each role will ideally have a shadow for succession planning. Those who
+have been a part of a previous event get first dibs on lead roles to maintain
+consistency.
 
-The Event Lead(s) is responsible for determining which core roles need to be filled at the start of planning for each event. After initial core team meetings, each lead will determine how to best staff for that function and fill in other roles. Each role will ideally have a shadow for succession planning. Those who have been a part of a previous event get first dibs on lead roles to maintain consistency.
+Time Commitment:
+- 1-8 hours per week leading up to the event
```

Comment:
It might make sense to break this up somewhat and give a rolling prediction. IIRC the release rolebooks do this.

Week 1-4, 6-8 hours
Week 5-8, 1-3 hours
Week 9-12, 6-8 hours

Only problem here is we don't have a set period of time. So it might make sense to call out "Early and Late in the process more time will be required. 

Might just be babbling, what are your thoughts?